x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/description_step/actions_description.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { startCase } from 'lodash/fp';
+import { AlertAction } from '../../../../../../../../../plugins/alerting/common';
+
+const ActionsDescription = ({ actions }: { actions: AlertAction[] }) => {
+  if (!actions.length) return null;
+
+  return (
+    <ul>
+      {actions.map((action, index) => (
+        <li key={index}>{getActionTypeName(action.actionTypeId)}</li>
```

Comment:
Maybe we should use the `actionTypeId` for the `key`, here? Is there a field unique to the action?

Reply:
you can have multiple actions of the same type, so in that case, `actionTypeId` will not be unique